src/main/java/frc/robot/Robot.java

```diff
@@ -33,7 +34,8 @@ public void robotInit() {
 
   @Override
   public void configureButtonBindings() {
-    DoubleSupplier trigger = driveController::getTriggers;
+    DoubleSupplier trigger = copilotController::getTriggers;
+    DoubleSupplier climberJoystick = copilotController::getLeftX;
```

Comment:
This isn't really necessary, but if you want to have it stored in a variable like this the name should be more specific.